packages/app/src/specs/banners/ConnectProjectBanner.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <TrackedBanner
+    :banner-id="BannerIds.ACI_082022_CONNECT_PROJECT"
+    :model-value="modelValue"
+    data-cy="connect-project-banner"
+    status="info"
+    :title="t('specPage.banners.connectProject.title')"
+    class="mb-16px"
+    :icon="ConnectIcon"
+    dismissible
+    @update:model-value="value => emit('update:modelValue', value)"
+  >
+    <p class="mb-24px">
+      {{ t('specPage.banners.connectProject.content') }}
+    </p>
+
+    <Button
+      :prefix-icon="ConnectIcon"
+      class="mt-24px"
+      data-cy="connect-project-button"
+      @click="handleButtonClick"
+    >
+      {{ t('specPage.banners.connectProject.buttonLabel') }}
+    </Button>
+
+    <CloudConnectModals
+      v-if="isProjectConnectOpen && cloudModalsQuery.data.value"
+      :gql="cloudModalsQuery.data.value"
+      @cancel="handleModalClose"
+      @success="handleModalClose"
```

Comment:
Nice, I really like the `handleXXX` or `onXXX` for event handles. Not sure if we can lint for this, but I am a fan of this convention.